File: src/main/java/dev/sky_lock/pocketlifevehicle/packet/UpdateSignPacket.java
package dev.sky_lock.pocketlifevehicle.packet;

import com.comphenix.protocol.events.PacketContainer;

/**
 * @author sky_lock
 */

public class UpdateSignPacket extends ClientPacket {

    public UpdateSignPacket(PacketContainer packet) {
        super(packet);
    }

    public String[] getLines() {
        return getPacket().getStringArrays().read(0);
    }
}

File: src/main/kotlin/dev/sky_lock/pocketlifevehicle/packet/UpdateSignPacket.kt
package dev.sky_lock.pocketlifevehicle.packet

import com.comphenix.protocol.events.PacketContainer

/**
 * @author sky_lock
 */
class UpdateSignPacket(packet: PacketContainer) : ClientPacket(packet) {
    val lines: Array<String>
        get() = packet.stringArrays.read(0)
}